### PR TITLE
Dropdown Secondary Button

### LINKED ISSUE #3932: Update Button in Spark's Information Dropdown Input
**User Story:**

**Notes:**

**AC**:
https://sparkdesignsystem.com/using-spark/components/dropdown
- Informational dropdown will use Sparks Secondary button

Current State:
image

### PR DESCRIPTION
**Pull request by guern1kn**
## What does this PR do?
Update the Informational Dropdown to use the secondary button.

### Associated Issue
Fixes #3932 

### Code
 - [x] Build Component in HTML
 - [x] Build Component in Angular
 - [x] Build Component in React
 - [x] Unit Testing in HTML with `npm run test` in `html/` (100% coverage, 100% passing)
 - [x] Unit Testing in Angular with `npm run test` in `angular/` (100% coverage, 100% passing)
 - [x] Unit Testing in React with `npm run test` in `react/` (100% coverage, 100% passing)

### Browser Testing (current version and 1 prior)
  - [x] Google Chrome
  - [x] Google Chrome (Mobile)
  - [x] Mozilla Firefox
  - [x] Mozilla Firefox (Mobile)
  - [x] Microsoft Internet Explorer 11 (only this specific version of IE)
  - [x] Microsoft Edge
  - [x] Apple Safari
  - [ ] Apple Safari (Mobile)

### Deploy Preview Reviewed and Approved
 - [ ] Design Team

### CODE REVIEW

File: react/src/components/dropdown/SprkDropdown.js [hunk 1]
```diff
@@ -203,7 +205,12 @@ class SprkDropdown extends Component {
           <div className={classNames('sprk-c-Dropdown', additionalClasses)}>
             {heading !== '' && (
               <div className="sprk-c-Dropdown__header">
-                <h2 className="sprk-c-Dropdown__title">{heading}</h2>
+                <SprkHeading
+                  element="h2"
```
Comment (afebbraro) [inline react/src/components/dropdown/SprkDropdown.js:209]: Does this need a variant? Without it, the heading component isn't doing anything.

File: react/src/components/dropdown/SprkDropdown.js [hunk 2]
```diff
@@ -266,16 +273,18 @@ class SprkDropdown extends Component {
                           }}
                           {...choicesRest}
                         >
-                          <p className="sprk-b-TypeBodyOne">{content.title}</p>
+                          <SprkText element="p" variant="bodyOne">
+                            {content.title}
+                          </SprkText>
                           {content.infoLine1 && (
-                            <p className="sprk-b-TypeBodyTwo">
+                            <SprkText element="p" variant="bodyTwo">
```
Comment (afebbraro) [inline react/src/components/dropdown/SprkDropdown.js:280]: default is set to p so dont need to redeclare it twice

File: react/src/components/dropdown/SprkDropdown.js (hunk 1 above)
Comment (guern1kn) [reply react/src/components/dropdown/SprkDropdown.js:209]: Looks like this was out of sync with html & angular and needs to have the bodyTwo class added. Will get it updated.

**Review by RVMendoza — APPROVED** [review]
tenor-195317194

**Review by afebbraro — APPROVED** [review]

**Comment by afebbraro** [comment]
Looks good, thanks!

**Pull request merged**